sale_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# © 2015-17 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
```

Comment:
see https://github.com/OCA/operating-unit/pull/152 first as most of the remarks apply